benchmarks/bench_simul.c

```diff
@@ -396,6 +400,13 @@ main(int argc, const char **argv)
 		printf("\n");
 	}
 
+	lotta_zeroes = mmap(NULL, max_size, PROT_READ,
```

Comment:
touch all these pages, otherwise we will measure the cost of page fault handler in memcpy